src/styles/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 198px; // including border
+  height: 406px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    padding: 32px 19px 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    color: $card-title-color;
+    padding-bottom: 4px;
+    padding-left: 16px;
+    padding-right: 16px;
+    max-width: 250px;
```

Comment:
The max-width for the element `.card__title` exceeds the width of the `.card` itself, which is set to 198px. This could lead to unexpected layout issues. Consider adjusting the max-width to be equal to or less than the width of the card.